src/towncrier/test/test_project.py

```diff
@@ -36,3 +37,24 @@ def test_tuple(self):
 
         version = get_version(temp, "mytestproja")
         self.assertEqual(version, "1.3.12")
+
+
+class InvocationTests(TestCase):
+    def test_dash_m(self):
+        """
+        `python -m towncrier` works.
+        """
+        temp = self.mktemp()
+        new_dir = os.path.join(temp, "dashm")
+        os.makedirs(new_dir)
+        orig_dir = os.getcwd()
+        try:
+            os.chdir(new_dir)
+            with open("pyproject.toml", "w") as f:
+                f.write('[tool.towncrier]\n' 'directory = "news"\n')
+            os.makedirs("news")
+            out = check_output(["python", "-m", "towncrier", "--help"])
```

Comment:
Probably should use `sys.executable` instead of `'python'` here, to avoid relying on PATH

Reply:
...hmm I didn't know this would show up as a "review", my intention was just to comment.